helix-core/src/main/java/org/apache/helix/monitoring/mbeans/WorkflowMonitor.java

```diff
@@ -142,12 +113,72 @@ public void resetGauges() {
    */
   public void updateWorkflowGauges(TaskState current) {
     if (current == null || current.equals(TaskState.NOT_STARTED)) {
-      _queuedWorkflowGauge++;
+      incrementSimpleDynamicMetric(_queuedWorkflowGauge, 1);
```

Comment:
Not for this PR, but this method seems to be a better fit in the SimpleDynamicMetric class